lib/openstudio-standards/prototypes/common/objects/Prototype.utilities.rb

```diff
@@ -367,6 +367,22 @@ def cop_no_fan_to_eer(cop, capacity_w = nil)
     return eer
   end
 
+  # Convert from IEER to COP (no fan)
+  #
+  # @note IEER is a weighted-average efficiency metrics at different load percentages, operataional and environemental conditions
+  # @note IEER should be modeled by using performance curves that match a targeted efficiency values
+  # @note This method estimates what a reasonable full load rated EER would be for a targeted IEER value
+  # @note The regression used in this method is based on a survey of over 1,000 rated AHRI units with IEER ranging from 11.8 to 25.6
+  # @todo Implement methods to handle IEER modeling
+  #
+  # @param ieer [Double] Energy Efficiency Ratio (EER)
+  # @return [Double] Coefficient of Performance (COP)
+  def ieer_to_cop_no_fan(ieer)
+    eer = 0.0183 * ieer * ieer - 0.4552 * ieer + 13.21
+
+    return eer_to_cop_no_fan(eer)
+  end
+
```

Comment:
New method to convert an IEER to a COP no fan. Note that this is just to estimate what would a reasonable full load gross COP would be for a specific IEER value. IEER-specific performance curves should then be applied. [Copper](https://github.com/pnnl/copper/tree/develop) could be used when it supports IEER rated equipment.